cmd/av/stack_prev.go

```diff
@@ -12,12 +12,17 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var stackPrevFlags struct {
+	// should we go to the first
+	First bool
+}
+
 var stackPrevCmd = &cobra.Command{
-	Use:   "prev <n>",
+	Use:   "prev <n> or prev --first",
 	Short: "checkout the previous branch in the stack",
 	RunE: func(cmd *cobra.Command, args []string) error {
 		var n int = 1
-		if len(args) == 1 {
+		if len(args) == 1 && !stackPrevFlags.First {
```

Comment:
Again, I think it would be better to just determine the target branch as two distinct branches rather than having them be interlocked like this.